src/RayMarchedAtlasVolume.ts

```diff
@@ -173,10 +179,27 @@ export default class RayMarchedAtlasVolume implements VolumeRenderImpl {
 
     if (dirtyFlags & SettingsFlags.MASK) {
       this.setUniform("maskAlpha", this.settings.maskAlpha);
-      this.setUniform("maskAlpha", this.settings.maskAlpha);
     }
   }
 
+  public updateVolumeDimensions(): void {
+    const { normalizedPhysicalSize, contentSize } = this.volume;
+    // Set offset
+    this.geometryMesh.position.copy(this.volume.getContentCenter());
+    // Set scale
+    this.geometryMesh.scale.copy(contentSize).multiply(normalizedPhysicalSize);
+    this.setUniform("volumeScale", normalizedPhysicalSize);
+    this.boxHelper.box.set(
+      normalizedPhysicalSize.clone().multiplyScalar(-0.5),
+      normalizedPhysicalSize.clone().multiplyScalar(0.5)
+    );
+    // TODO do tickmarks need to be regenerated here to remain accurate?
```

Comment:
if the answer is "yes" then we have a bug?

Reply:
I thought about this since leaving this comment and I think for a well-formed dataset the answer is no. If a load changes the size of the subregion, that should have no impact on the size of the bounding box. If a load changes the size of the full volume, it should only be to switch scale levels. In that case, it should still hold the physical size and aspect ratio of the volume constant within a rounding error for all but the smallest and/or silliest datasets.